draft-ietf-jsonpath-base.md

```diff
@@ -1391,22 +1391,31 @@ Note that `..` on its own is not a valid selector.
 #### Semantics
 {: unnumbered}
 
-A `descendant-selector` selects certain descendants of a node:
+A descendant selector selects zero or more descendants of a node.
 
-* the `..<name>` form (and the `..[<index>]` form where `<index>` is a `quoted-member-name`) selects those descendants that are member values of an object with the given member name.
-* the `..[<index>]` form, where `<index>` is an `element-index`, selects those descendants that are array elements with the given index.
-* the `..[<slice-index>]` form selects those descendants that are array elements selected by the given slice.
-* the `..[<filter>]` form selects those descendants that are array elements or object values selected by the given filter.
-* the `..[*]` and `..*` forms select all the descendants.
-
-An _array-sequenced preorder_ of the descendants of a node is a sequence of all the descendants in which:
+A nodelist enumerating the descendants is known as a _descendant nodelist_ when:
 
 * nodes of any array appear in array order,
 * nodes appear immediately before all their descendants.
 
-This definition does not stipulate the order in which the children of an object appear, since JSON objects are unordered.
+This definition does not stipulate the order in which the children of an object appear, since
+JSON objects are unordered.
```

Comment:
Slightly more correct I think: The order of the members in a JSON object is not information-bearing.

Reply:
The sentence "The order of the members in a JSON object is not information-bearing." is a bit tricky because it refers to "The order of the members in a JSON object" as if that was somehow well defined. Also, I'm not sure how many people would understand the sentence because the term "information-bearing" is a little unusual. I think I'll stick with the informal language for now.